### PR TITLE
Update endpoint-slices.md

### PR DESCRIPTION
**Pull request by YunWZ**
fix translation errors about "Distribution of EndpointSlices" section.

### CODE REVIEW

**Comment by windsonsea** [comment]
Hi, we do not accept part of changes when much inconsistency exists. Please try to run this command to check difference with en:
```
./scripts/lsync.sh content/zh-cn/docs/concepts/services-networking/endpoint-slices.md
```
The output may be similar to:

image

**Comment by YunWZ** [comment]
> Hi, we do not accept part of changes when much inconsistency exists. Please try to run this command to check difference with en:
> 
> ```
> ./scripts/lsync.sh content/zh-cn/docs/concepts/services-networking/endpoint-slices.md
> ```
> 
> The output may be similar to:
> 
> image

the command executed result:
```bash
 $ ./scripts/lsync.sh content/zh-cn/docs/concepts/services-networking/endpoint-slices.md
content/zh-cn/docs/concepts/services-networking/endpoint-slices.md is still in sync
```
Is it ok?

**Comment by windsonsea** [comment]
Run the command in the `main` branch to find differences, and modify all zh text that is different from en.

**Comment by YunWZ** [comment]
> Run the command in the `main` branch to find differences, and modify all zh text that is different from en.
> 
> If not, you may need to check it line by line manually.

@windsonsea @tengqm Hi， I have merged the latest content of the main branch and modified the Chinese translation of the corresponding content.

File: content/zh-cn/docs/concepts/services-networking/endpoint-slices.md [hunk 1]
```diff
@@ -90,7 +59,7 @@ EndpointSlice 通过唯一的协议、端口号和 Service 名称将网络端点
 EndpointSlice 的名称必须是合法的
 [DNS 子域名](/zh-cn/docs/concepts/overview/working-with-objects/names#dns-subdomain-names)。
 
-例如，下面是 Kubernetes Service `example` 的 EndpointSlice 资源示例。
+例如，下面是 Kubernetes Service `example` 的 EndpointSlice 对象示例。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/endpoint-slices.md:62]:
```suggestion
例如，下面是 Kubernetes Service `example` 所拥有的 EndpointSlice 对象示例。
```

File: content/zh-cn/docs/concepts/services-networking/endpoint-slices.md [hunk 2]
```diff
@@ -152,6 +123,9 @@ EndpointSlice 支持三种地址类型：
 * IPv6
 * FQDN (完全合格的域名)
 
+每个 `EndpointSlice` 对象代表一个特定的 IP 地址类型。 如果你有 一个支持 IPv4 和 IPv6 的服务，
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/endpoint-slices.md:126]:
```suggestion
每个 `EndpointSlice` 对象代表一个特定的 IP 地址类型。如果你有一个支持 IPv4 和 IPv6 的服务，
```

File: content/zh-cn/docs/concepts/services-networking/endpoint-slices.md [hunk 3]
```diff
@@ -443,20 +417,82 @@ EndpointSlice 对象。Deployment 的滚动更新为重新为 EndpointSlice
...
+<!--
+Clients of the EndpointSlice API must be able to handle the situation where
+a particular endpoint address appears in more than one slice.
+
+You can find a reference implementation for how to perform this endpoint deduplication
+as part of the `EndpointSliceCache` code within `kube-proxy`.
+-->
+EndpointSlice API 的客户端必须能够处理特定端点地址出现在多个 EndpointSlice 中的情况。
+
+您可以在 `kube-proxy` 中的 `EndpointSliceCache` 代码中找到有关如何执行端点重复数据删除的参考实现。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/endpoint-slices.md:440]:
```suggestion
你可以在 `kube-proxy` 中的 `EndpointSliceCache` 代码中找到有关如何执行这个端点去重的参考实现。
```

File: content/zh-cn/docs/concepts/services-networking/endpoint-slices.md [hunk 4]
```diff
@@ -443,20 +417,82 @@ EndpointSlice 对象。Deployment 的滚动更新为重新为 EndpointSlice
...
+
+The original Endpoints API provided a simple and straightforward way of
+tracking network endpoints in Kubernetes. As Kubernetes clusters
+and {{< glossary_tooltip text="Services" term_id="service" >}} grew to handle
+more traffic and to send more traffic to more backend Pods, the
+limitations of that original API became more visible.
+Most notably, those included challenges with scaling to larger numbers of
+network endpoints.
+-->
+## 与 Endpoints 的比较 {#motivation}
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/endpoint-slices.md:455]:
```suggestion
## 与 Endpoints 的比较 {#motivation}

```

File: content/zh-cn/docs/concepts/services-networking/endpoint-slices.md [hunk 5]
```diff
@@ -443,20 +417,82 @@ EndpointSlice 对象。Deployment 的滚动更新为重新为 EndpointSlice
...
+更新都意味着 Kubernetes 集群组件之间的大量流量（在控制平面内， 以及在节点和 API 服务器之间）。
+这种额外的流量在 CPU 使用方面也有成本。
+
+<!--
+With EndpointSlices, adding or removing a single Pod triggers the same _number_
+of updates to clients that are watching for changes, but the size of those
+update message is much smaller at large scale.
+-->
+使用 EndpointSlices，添加或删除单个 Pod 会触发相同数量的更新到正在监视更改的客户端，但这些更新
+消息的大小在大规模时要小得多。（译者注：与使用 Endpoints 的方式相比）
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/endpoint-slices.md:489]:
```suggestion
使用 EndpointSlices，添加或移除单个 Pod 对于正监视变更的客户端会触发相同数量的更新，
但这些更新消息的大小在大规模使用时要小得多。
```
咦，`译者注`有必要加吗

**Comment by windsonsea** [comment]
Squash your commits into one please.

**Comment by YunWZ** [comment]
> Squash your commits into one please.

done.

File: content/zh-cn/docs/concepts/services-networking/endpoint-slices.md [hunk 6]
```diff
@@ -443,20 +417,82 @@ EndpointSlice 对象。Deployment 的滚动更新为重新为 EndpointSlice
...
+
+<!--
+Since all network endpoints for a Service were stored in a single Endpoints
+object, those Endpoints objects could get quite large. For Services that stayed
+stable (the same set of endpoints over a long period of time) the impact was
+less noticeable; even then, some use cases of Kubernetes weren't well served.
+-->
+由于任一 Service 的所有网络端点都保存在同一个 Endpoints 对象中， 这些 Endpoints 对象可能变
+得非常巨大。 对于保持稳定的服务（长时间使用同一组端点），影响不太明显； 即便如此， Kubernetes 的
+一些使用场景也没有得到很好的服务。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/endpoint-slices.md:468]:
```suggestion
由于任一 Service 的所有网络端点都保存在同一个 Endpoints 对象中，这些 Endpoints
对象可能变得非常巨大。对于保持稳定的服务（长时间使用同一组端点），影响不太明显；
 即便如此，Kubernetes 的一些使用场景也没有得到很好的服务。
```

File: content/zh-cn/docs/concepts/services-networking/endpoint-slices.md [hunk 7]
```diff
@@ -443,20 +417,82 @@ EndpointSlice 对象。Deployment 的滚动更新为重新为 EndpointSlice
...
+When a Service had a lot of backend endpoints and the workload was either
+ scaling frequently, or rolling out new changes frequently, each update to
+the single Endpoints object for that Service meant a lot of traffic between
+Kubernetes cluster components (within the control plane, and also between
+nodes and the API server). This extra traffic also had a cost in terms of
+CPU use.
+-->
+当服务有很多后端端点并且该工作负载频繁扩缩或上线新更改时，对该服务的单个 Endpoints 对象的每次
+更新都意味着 Kubernetes 集群组件之间的大量流量（在控制平面内， 以及在节点和 API 服务器之间）。
+这种额外的流量在 CPU 使用方面也有成本。
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/services-networking/endpoint-slices.md:481]:
```suggestion
当服务有很多后端端点并且该工作负载频繁扩缩或上线新更改时，对该服务的单个 Endpoints
对象的每次更新都意味着（在控制平面内以及在节点和 API 服务器之间）Kubernetes 集群组件之间会出现大量流量。
这种额外的流量在 CPU 使用方面也有成本。
```

**Review by windsonsea — COMMENTED** [review]
注意空格和换行问题。

1. 中英之间保留 1 个且仅有 1 个空格
2. 长句时在中英、数字、标点附近适当折行，不要在中文字之间折行

**Comment by Sea-n** [comment]
Hey @YunWZ,

It's been a while since the last update, would you like to address issues in the previous comments so that we can move forward?

**Comment by YunWZ** [comment]
> Hey @YunWZ,
> 
> It's been a while since the last update, would you like to address issues in the previous comments so that we can move forward?

done.

File: content/zh-cn/docs/concepts/services-networking/endpoint-slices.md [hunk 8]
```diff
@@ -152,6 +123,9 @@ EndpointSlice 支持三种地址类型：
 * IPv6
 * FQDN (完全合格的域名)
 
+每个 `EndpointSlice` 对象代表一个特定的 IP 地址类型。如果你有一个支持 IPv4 和 IPv6 的服务，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/endpoint-slices.md:126]:
```suggestion
每个 `EndpointSlice` 对象代表一个特定的 IP 地址类型。如果你有一个支持 IPv4 和 IPv6 的 Service，
```

File: content/zh-cn/docs/concepts/services-networking/endpoint-slices.md [hunk 9]
```diff
@@ -443,20 +417,82 @@ EndpointSlice 对象。Deployment 的滚动更新为重新为 EndpointSlice
...
+and {{< glossary_tooltip text="Services" term_id="service" >}} grew to handle
+more traffic and to send more traffic to more backend Pods, the
+limitations of that original API became more visible.
+Most notably, those included challenges with scaling to larger numbers of
+network endpoints.
+-->
+## 与 Endpoints 的比较 {#motivation}
+原来的 Endpoints API 提供了在 Kubernetes 中跟踪网络端点的一种简单而直接的方法。随着 Kubernetes
+集群和{{< glossary_tooltip text="服务" term_id="service" >}}逐渐开始为更多的后端 Pod 处理和发送请求，
+原来的 API 的局限性变得越来越明显。最重要的是那些因为要处理大量网络端点而带来的挑战。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/endpoint-slices.md:458]:
```suggestion
原来的 API 的局限性变得越来越明显。最明显的是那些因为要处理大量网络端点而带来的挑战。
```

File: content/zh-cn/docs/concepts/services-networking/endpoint-slices.md [hunk 10]
```diff
@@ -443,20 +417,82 @@ EndpointSlice 对象。Deployment 的滚动更新为重新为 EndpointSlice
...
+
+<!--
+Since all network endpoints for a Service were stored in a single Endpoints
+object, those Endpoints objects could get quite large. For Services that stayed
+stable (the same set of endpoints over a long period of time) the impact was
+less noticeable; even then, some use cases of Kubernetes weren't well served.
+-->
+由于任一 Service 的所有网络端点都保存在同一个 Endpoints 对象中，这些 Endpoints
+对象可能变得非常巨大。对于保持稳定的服务（长时间使用同一组端点），影响不太明显；
+ 即便如此，Kubernetes 的一些使用场景也没有得到很好的服务。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/endpoint-slices.md:468]:
```suggestion
即便如此，Kubernetes 的一些使用场景也没有得到很好的服务。
```

File: content/zh-cn/docs/concepts/services-networking/endpoint-slices.md [hunk 11]
```diff
@@ -443,20 +417,82 @@ EndpointSlice 对象。Deployment 的滚动更新为重新为 EndpointSlice
...
+
+<!--
+When a Service had a lot of backend endpoints and the workload was either
+ scaling frequently, or rolling out new changes frequently, each update to
+the single Endpoints object for that Service meant a lot of traffic between
+Kubernetes cluster components (within the control plane, and also between
+nodes and the API server). This extra traffic also had a cost in terms of
+CPU use.
+-->
+当服务有很多后端端点并且该工作负载频繁扩缩或上线新更改时，对该服务的单个 Endpoints
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/endpoint-slices.md:479]:
```suggestion
当某 Service 存在很多后端端点并且该工作负载频繁扩缩或上线新更改时，对该 Service 的单个 Endpoints
```

File: content/zh-cn/docs/concepts/services-networking/endpoint-slices.md [hunk 12]
```diff
@@ -443,20 +417,82 @@ EndpointSlice 对象。Deployment 的滚动更新为重新为 EndpointSlice
...
+对象的每次更新都意味着（在控制平面内以及在节点和 API 服务器之间）Kubernetes 集群组件之间会出现大量流量。
+这种额外的流量在 CPU 使用方面也有成本。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/endpoint-slices.md:481]:
```suggestion
这种额外的流量在 CPU 使用方面也有开销。
```

File: content/zh-cn/docs/concepts/services-networking/endpoint-slices.md [hunk 13]
```diff
@@ -443,20 +417,82 @@ EndpointSlice 对象。Deployment 的滚动更新为重新为 EndpointSlice
...
+
+<!--
+With EndpointSlices, adding or removing a single Pod triggers the same _number_
+of updates to clients that are watching for changes, but the size of those
+update message is much smaller at large scale.
+-->
+使用 EndpointSlices，添加或移除单个 Pod 对于正监视变更的客户端会触发相同数量的更新，
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/endpoint-slices.md:488]:
```suggestion
使用 EndpointSlices 时，添加或移除单个 Pod 对于正监视变更的客户端会触发相同数量的更新，
```

File: content/zh-cn/docs/concepts/services-networking/endpoint-slices.md [hunk 14]
```diff
@@ -443,20 +417,82 @@ EndpointSlice 对象。Deployment 的滚动更新为重新为 EndpointSlice
...
+但这些更新消息的大小在大规模使用时要小得多。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/services-networking/endpoint-slices.md:489]:
```suggestion
但这些更新消息的大小在大规模场景下要小得多。
```

**Comment by mengjiao-liu** [comment]
> Squash your commits into one please.

The commits are 8 now.

**Comment by tengqm** [comment]
Thanks
/lgtm
/approve

**Pull request merged**